ui/components/ui/contract-token-values/contract-token-values.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconCopy from '../icon/icon-copy';
+import IconBlockExplorer from '../icon/icon-block-explorer';
+import Box from '../box/box';
+import Tooltip from '../tooltip/tooltip';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Identicon from '../identicon/identicon.component';
+import Typography from '../typography/typography';
+import {
+  FONT_WEIGHT,
+  TYPOGRAPHY,
+  DISPLAY,
+  COLORS,
+  ALIGN_ITEMS,
+  JUSTIFY_CONTENT,
+} from '../../../helpers/constants/design-system';
+import Button from '../button';
+import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
+
+export default function ContractTokenValues({ address, tokenName }) {
+  const t = useI18nContext();
+  const [copied, handleCopy] = useCopyToClipboard();
+
+  return (
+    <Box
+      display={DISPLAY.FLEX}
+      alignItems={ALIGN_ITEMS.CENTER}
+      justifyContent={JUSTIFY_CONTENT.CENTER}
+      className="contract-token-values"
+    >
+      <Box marginRight={2}>
+        <Identicon address={address} diameter={24} />
+      </Box>
+      <Typography
+        variant={TYPOGRAPHY.H2}
+        fontWeight={FONT_WEIGHT.BOLD}
+        color={COLORS.TEXT_ALTERNATIVE}
+        marginTop={0}
+        marginBottom={0}
+      >
+        {tokenName}
+      </Typography>
+      <Box className="contract-token-values__copy-address">
+        <Tooltip
+          position="top"
+          title={copied ? t('copiedExclamation') : t('copyToClipboard')}
+        >
+          <Button
+            type="link"
+            className="contract-token-values__copy-address__button"
+            onClick={() => {
+              handleCopy(address);
+            }}
+          >
+            <IconCopy size={24} color="var(--color-icon-muted)" />
+          </Button>
+        </Tooltip>
+      </Box>
+      <Box className="contract-token-values__block-explorer">
+        <Tooltip position="top" title={t('openInBlockExplorer')}>
+          <Button
+            type="link"
+            className="contract-token-values__block-explorer__button"
+          >
+            <IconBlockExplorer size={16} color="var(--color-icon-muted)" />
```

Comment:
Once the svg has been updated you can change this to 24
```suggestion
            <IconBlockExplorer size={24} color="var(--color-icon-muted)" />
```

Reply:
Hi @georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/cfc287279869d91b7e1592d2166ecb5885fc9a17).